app/src/main/java/org/kiwix/kiwixmobile/zim_manager/ZimManageActivity.java

```diff
@@ -45,6 +46,7 @@
 import javax.inject.Inject;
 
 import static org.kiwix.kiwixmobile.utils.StyleUtils.dialogStyle;
+import static org.kiwix.kiwixmobile.zim_manager.library_view.LibraryFragment.DownloadServiceConnection.*;
```

Comment:
I am not sure we need these imports. You have just removed a method. Why add these ? May be you were trying out things but make sure to optimize imports before sending the PR.

Reply:
@srv-twry I wanted to ask about that. Could we like write a gradle script or something so that we don't need to remove those manually?

Reply:
Android studio has a shortcut ```ctrl + alt + o```. I haven't seen anyone writing the gradle script for it.